reference/configuration/framework.rst

```diff
@@ -1220,6 +1227,31 @@ dsn
 
 The DSN where to store the profiling information.
 
+rate_limiter
+~~~~~~~~~~~~
+
+.. _reference-rate-limiter-name:
+
+name
+""""
+
+**type**: ``prototype``
+
+Name of the rate limiter you want to create.
+
+lock_factory
+""""""""""""
+
+The service that is used to create a lock. The service has to implement the
+:class:`Symfony\\Component\\Lock\\LockFactoryInterface`.
```

Comment:
In 5.2 it should be `LockFactory`

But (hopefully) this is correct in 5.3: https://github.com/symfony/symfony/pull/39634

Reply:
We did this change while merging. Cheers!